source/chapters/user_interface.rst

```diff
@@ -113,8 +113,9 @@ visually by aligning the beats that appear in each waveform.
    Mixxx default skin (Latenight) - Parallel waveforms
 
 Depending on the skin, Mixxx displays either parallel waveforms (default) or
-separate waveforms. Select your preferred appearance in
-:menuselection:`Preferences --> Interface --> Skin`.
+separate waveforms. You can select your preferred appearance by clicking the 
+gear icon in the upper right corner. Then under Skin Settings, click/toggle 
+the Parallel Waveforms option to choose between parallel or separate waveform.
```

Comment:
Let's use the [`:menuselection:` role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-menuselection) for this and remove the repetition.

```suggestion
gear icon in the upper right corner, then toggling the :menuselection:`Skin Settings --> Parallel Waveforms` option.
```

Reply:
alright. let me implement this and Ronso's change. Thank you.